src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java

```diff
@@ -475,7 +484,10 @@ private void performUploadWithRetry(Executor executor, Storage service,
         return;
       } catch (IOException e) {
```

Comment:
Unfortunately, the precise semantics of multiple catch don't work for what we want to do with this exception.